<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingWindow</class>
 <widget class="QMainWindow" name="settingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btu1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>声音</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btu4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>地图大小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
